include/libopencm3/stm32/l0/rcc.h

```diff
@@ -505,32 +507,40 @@ enum rcc_periph_clken {
 	RCC_TIM21 = _REG_BIT(0x34, 2),
 	RCC_TIM22 = _REG_BIT(0x34, 5),
 	RCC_MIFI = _REG_BIT(0x34, 7),
+	RCC_FW = _REG_BIT(0x34, 7),
```

Comment:
Just used stm32l0x2 ref man bit name (FWEN) (in RCC_APB2ENR). That ref man does not contain the word 'mifare', just uses 'firewall'.